src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
+        lerp = function(start, stop, amt) {
```

Comment:
since we're already declaring lerp in other places, it'll be good to turn this into its own helper function that we require in, instead. It'll be worth creating a new file like `./src/Helpers/lerp.js` with content:
```
module.exports = function lerp(r,v1,v2) {
  return v1 + r*(v2-v1);
};
```
and then doing `lerp = require("../Helpers/lerp") here, and in https://github.com/processing-js/processing-js/blob/master/src/P5Functions/Math.js#L258 doing `p.lerp = require("../Helpers/lerp");`